### ISSUE TITLE
Incompatibility with Latest Version?

### ISSUE THREAD

**Issue by DrLongGhost** [pnuckowski/aioresponses#34]
I tried to use aioresponses version 0.1.2 with aiohttp==1.1.6 and was unable to get it to work. I suspect that maybe I am using an older version of your library or too new a version of aiohttp. Are you able to provide some assistance?

You can see the relevant section of my code in the stack trace below. I'm more or less following the same format as in your examples but it seems like the mock response that is being returned by aiohttp is not being accepted by aiohttp/ClientResponse?  Is there a newer version I should be using or something else the problem?

Thanks.

```self = <cheesesteak.src.tests.unit.test_classification.TestEcontextApiCalls testMethod=test_econtext_url_query>
mocked = <aioresponses.core.aioresponses object at 0x10269fcf8>

    @aioresponses()
    def test_econtext_url_query(self, mocked):
        # Set up mocks
        session = aiohttp.ClientSession()
        mocked.post('https://api.econtext.com/v2/classify/url', status=200, body='{"test": 1}')

        # Call the method
        loop = asyncio.get_event_loop()
>       json_response = loop.run_until_complete(session.post('https://api.econtext.com/v2/classify/url'))

src/tests/unit/test_classification.py:335:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/asyncio/base_events.py:337: in run_until_complete
    return future.result()
/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/asyncio/futures.py:274: in result
    raise self._exception
/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/asyncio/tasks.py:239: in _step
    result = coro.send(None)
/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/asyncio/coroutines.py:206: in coro
    res = func(*args, **kw)
../../.virtualenvs/cheesesteak/lib/python3.5/site-packages/aioresponses/core.py:154: in _request_mock
    response = self.match(method, url)
../../.virtualenvs/cheesesteak/lib/python3.5/site-packages/aioresponses/core.py:140: in match
    for i, r in enumerate(self._responses)
../../.virtualenvs/cheesesteak/lib/python3.5/site-packages/aioresponses/core.py:141: in <listcomp>
    if r.match(method, url)]
../../.virtualenvs/cheesesteak/lib/python3.5/site-packages/aioresponses/core.py:50: in build_response
    self.resp = ClientResponse(self.method, self.url)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <[AttributeError("'ClientResponse' object has no attribute '_url_obj'") raised in repr()] ClientResponse object at 0x10287aa90>
method = 'post', url = 'https://api.econtext.com/v2/classify/url'

    def __init__(self, method, url, *, writer=None, continue100=None,
                 timeout=5*60):
>       assert isinstance(url, URL)
E       AssertionError

../../.virtualenvs/cheesesteak/lib/python3.5/site-packages/aiohttp/client_reqrep.py:506: AssertionError```

**Comment by wandel** [comment]
Should hopefully be fixed in https://github.com/pnuckowski/aioresponses/pull/39 :)

**Comment by pnuckowski** [comment]
Closing. Already fixed in [0.1.3](https://github.com/pnuckowski/aioresponses/tree/0.1.3)
